src/commonMethods/commonFunctions.js

```diff
@@ -0,0 +1,296 @@
+            stream.end()
+            conn.end()
+            if (opts.cbWhenStarted !== true) cb(res)
+          })
+          .on('data', function(data) {
+            res += data
+          })
+          .stderr.on('data', function(data) {
+          })
+      })
+    })
+    .connect({
+      host: constants.host,
+      port: 22,
+      username: 'root',
+      password: 'root',
+    })
+
+  conn.on('timeout', function(e) {
+    throw new Error(`{opts.cmd}: Timeout while connecting to ${constants.host}`)
+  })
+
+  conn.on('error', function(err) {
+    throw new Error(`${opts.cmd}:  ${err}`)
+  })
+}
+
+/**
+ * This function calculates the average of FPS samples collected in fetchFPS function
+ * @returns {results}
+ */
+export const calcAvgFPS = function() {
+  let sum = 0
+  let average
+  let samples = this.$data.read('samples')
+  for (let i = 0; i < samples.length; i++) {
+    sum += samples[i]
+  }
+  average = sum / samples.length
+  average = average.toFixed(2)
+  this.$data.write('average', average)
+}
+
+/**
+ * This function is used to get screenshot
+ * @returns {Promise<void>}
+ */
+export const screenshot = async function() {
+  let url = `http://${constants.host}:80/Service/Snapshot/Capture?${moment().valueOf()}`
+  // create a new promise inside of the async function
+  let bufferData = new Promise((resolve, reject) => {
+    _http
+      .get(url, function(res) {
+        if (res.headers['content-length'] === undefined)
+          this.$log(
+            'Framework did not return a content-length! This will slow down the screenshot module.'
+          )
+        var buffers = []
+        var imageSize = res.headers['content-length']
+
+        res.on('data', function(chunk) {
+          buffers.push(Buffer.from(chunk))
+        })
+
+        res.on('end', function() {
+          return resolve(Buffer.concat(buffers, parseInt(imageSize)))
+        })
+      })
+      .on('error', function(e) {
+        return reject(e)
+      })
+  })
+
+  // wait for the promise to resolve
+  let result = await bufferData
+  this.$data.write('screenshotResult', result)
+}
+
+/**
+ * This function is used to kill the process
+ * @param process
+ */
+export const killProcess = function(process) {
+  exec({ cmd: `killall -9 ${process}` }, err => {
+    return true
+  })
+}
+
+/**
+ * This function is used to start the framework
+ */
+export const startFramework = function() {
+  exec({ cmd: 'nohup WPEFramework WPEProcess -b &', cbWhenStarted: true }, err => {
+    return true
+  })
+}
+
+/**
+ * This function is used to restart the framework
+ */
+export const restartFramework = function() {
+  return this.$sequence([
+    () => killProcess('WPEFramework'),
+    () => killProcess('WPEProcess'),
+    () => startFramework(),
+  ])
+}
+
+/**
+ * This function performs below operations on WebKitBrowser Plugin
+ *  - Deactivate
+ *  - Activate
+ *  - Resume
+ */
+export const webKitBrowserOps = function() {
```

Comment:
Suggest to rename this for a more readable name, e.g. webKitBrowserRestart or RestartAndResume

Reply:
Renamed function name to a readable name as 'webKitBrowserStartAndResume'